keyboards/smithrune/iron160/iron160_h/config.h

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2022 Gondolindrim
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#define RGB_DI_PIN B15
+#define RGBLED_NUM 1
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* Defining RGB default brightness to zero */
+#define RGBLIGHT_DEFAULT_VAL 0
```

Comment:
This file can be removed with the above suggestion. (settings RGB light brightness to 0 seems like it'll be confusing for users)

Reply:
This is intentional -- we want the RGBs turned off by default.